experiment/simulation/ring_counter.html

```diff
@@ -37,100 +37,89 @@
         </div>
         
         <div class="columns is-centered">
-            <div class="column">
-                <div class="v-instruction-container instructions-box">
-                    <div class="v-instruction-title" id="task-title">Ring Counter</div>
-                    <div class="v-instruction-text" id="task-description">
-                        Simulation of a Ring Counter built using D flip-flops.
-                    </div>
-                    <p>Start- Starts the simulation after the inputs are properly selected. Default speed is 1x. Select the speed through dropdown menu</p>
-                    <p>Reset - Resets the simulation</p>
-                    <p>Click on the green input bits to set their values.</p>
-                    <p>This is a Ring Counter.</p>
-                    <p>There is a clock and ori which can be started by the user but it's cycle cannot be controlled.</p>
-                    <p>There are 3 outputs QA,QB,QC </p>
-                </div>
-            </div>
-        </div>
+			<div class="column">
```

Comment:
Please fix indentation

Reply:
We have fixed indentation but it might be a github issue that the indentaion doesn't come out properly